src/components/Dashboard/Tiles/Apps/AppPing.tsx

```diff
@@ -16,17 +17,55 @@ export const AppPing = ({ app }: AppPingProps) => {
   const active =
     (config?.settings.customization.layout.enabledPing && app.network.enabledStatusChecker) ??
     false;
+
+  console.log(
+    `app ping for ${app.id} active: ${active ? 'yes' : 'no'} -> [${
+      app.network.enabledStatusChecker
+    } && ${config?.settings.customization.layout.enabledPing}]`
+  );
+
   const { data, isLoading, error } = usePingQuery(app, active);
 
   const isOnline = data?.state === 'online';
 
   if (!active) return null;
 
+  const disablePulse = config?.settings.customization.accessibility?.disablePingPulse ?? false;
+  const replaceDotWithIcon =
+    config?.settings.customization.accessibility?.replacePingDotsWithIcons ?? false;
+
+  if (disablePulse) {
```

Comment:
Seems a little repetitive here, maybe just disable the animation of the motion.div. Additionally you should use the indicator ping as well on the other option when not disabled.